app/questionnaire/navigator.py

```diff
@@ -59,6 +61,14 @@ def build_path(blocks, block_id, answers, path):
     return path
 
 
+def get_navigator(schema_json):
```

Comment:
We might not be able to. I put the get_questionnaire_manager in questionnaire_manager.py because of a circular import dependency i was facing